projects/packages/jetpack-mu-wpcom/src/features/help-center/class-help-center.php

```diff
@@ -52,11 +52,22 @@ private static function is_proxied() {
 	/**
 	 * Creates instance.
 	 *
-	 * @return \A8C\FSE\Help_Center
+	 * @return \A8C\FSE\Help_Center|void
 	 */
 	public static function init() {
+		if ( defined( 'MU_WPCOM_HELP_CENTER' ) && MU_WPCOM_HELP_CENTER ) {
+			return;
+		}
+
+		$request_uri = isset( $_SERVER['REQUEST_URI'] ) ? sanitize_text_field( wp_unslash( $_SERVER['REQUEST_URI'] ) ) : '';
+
+		if ( str_contains( $request_uri, 'wp-content/plugins/gutenberg-core' ) || str_contains( $request_uri, 'preview=true' ) ) {
+			return;
+		}
+
 		if ( self::$instance === null ) {
 			self::$instance = new self();
+			define( 'MU_WPCOM_HELP_CENTER', true );
 		}
```

Comment:
If you take the suggestion above, then this becomes redundant.
```suggestion
		self::$instance = new self();
```